analyses/bigquery_shim/bigquery_shim/dashboard.py

```diff
@@ -25,6 +25,58 @@ def fetch_results(
         channel_filter = "normalized_channel = '{}' AND".format(channel)
 
     query = """
+    CREATE TEMP FUNCTION hist_to_json(input STRING) AS (
```

Comment:
I wonder if this UDF is generally useful to be added to `mozfun`. Then it would also be easier to add some unit tests.